src/graphql/admin/root/mutation/cold-storage-rebalance-to-hot-wallet.ts

```diff
@@ -0,0 +1,45 @@
+import { GT } from "@graphql/index"
+import PsbtDetailPayload from "@graphql/admin/types/payload/psbt-detail"
+import { ColdStorage } from "@app"
+import { mapError } from "@graphql/error-map"
+import SatAmount from "@graphql/types/scalar/sat-amount"
+import TargetConfirmations from "@graphql/types/scalar/target-confirmations"
+
+const ColdStorageRebalanceToHotWalletInput = new GT.Input({
+  name: "ColdStorageRebalanceToHotWalletInput",
+  fields: () => ({
+    walletName: { type: GT.NonNull(GT.String) },
+    amount: { type: GT.NonNull(SatAmount) },
+    targetConfirmations: { type: TargetConfirmations, defaultValue: 1 },
+  }),
+})
+
+const ColdStorageRebalanceToHotWalletMutation = GT.Field({
+  type: GT.NonNull(PsbtDetailPayload),
+  args: {
+    input: { type: GT.NonNull(ColdStorageRebalanceToHotWalletInput) },
+  },
+  resolve: async (_, args) => {
+    const { walletName, amount, targetConfirmations } = args.input
+    for (const input of [walletName, amount, targetConfirmations]) {
+      if (input instanceof Error) {
+        return { errors: [{ message: input.message }] }
```

Comment:
We get the type error here as well for this array object. I tested in the main schema and we use the exact `IError` interface and mutation pattern but don't have this issue. Really confused why this is different in the admin schema

Reply:
Found it https://github.com/GaloyMoney/galoy/pull/986/commits/fba0f1fbcc64421c2102e7987c69698a4972cf6d